srcpkgs/libportal/template

```diff
@@ -1,24 +1,28 @@
 # Template file for 'libportal'
 pkgname=libportal
-version=0.6
+version=0.7.1
 revision=1
 build_style=meson
-configure_args="$(vopt_bool gtk_doc docs) $(vopt_bool gir vapi) -Dbackends=gtk3,gtk4"
+configure_args="$(vopt_bool gtk_doc docs) $(vopt_bool gir vapi)
+ -Dbackend-gtk3=enabled -Dbackend-gtk4=enabled -Dbackend-qt5=enabled"
 hostmakedepends="pkg-config glib-devel gobject-introspection $(vopt_if gir vala)
  $(vopt_if gtk_doc gi-docgen)"
```

Comment:
```suggestion
hostmakedepends="pkg-config glib-devel gobject-introspection qt5-host-tools
 $(vopt_if gir vala) $(vopt_if gtk_doc gi-docgen)"
```

Reply:
`qt5-host-tools` is required for crossbuilding. i followed the logic of the original template. why is your placement more preferable?

Reply:
It is always needed, it just gets installed in the target sysroot by `qt5-tools-devel` in `makedepends`.

Reply:
thanks for the clarification!